.github/workflows/ci.yml

```diff
@@ -64,6 +64,7 @@ jobs:
         pip install https://github.com/pyro-ppl/funsor/archive/master.zip
         pip install jaxlib
         pip install jax
+        pip install tensorflow-cpu tfp-nightly
```

Comment:
Does TFP depend on tensorflow by default, or is that optional?

Reply:
The TF dependency is optional. If all you want is jax, `from tensorflow_probability.substrates import jax as tfp`

Reply:
Thanks @brianwa84! It is so helpful.

Reply:
@fehiepsi - can we just `pip install tensorflow_probability tfp-nightly` here?

Reply:
Yes, I think we can remove `tensorflow-cpu`. Let me remove it in the next commit to see if there is any issue happens.